### PR TITLE
Fixes oct21

### PR DESCRIPTION
**Pull request by RobertSchaffer1**
@mrakitin @JunAishima 

I'm not 100% sure who can review this, or even if that's required in this case. We (@erikfarquhar and myself) tested these fixes and approved of them.

### CODE REVIEW

**Comment by mrakitin** [comment]
@RobertSchaffer1, I'll review it on Monday, but maybe it'd make sense to meet at the beamline and check the problem and solution there?

File: startup/99-gui-ht.py [hunk 1]
```diff
@@ -716,8 +717,15 @@ def main_plan(file_name):
             if file_name is None:
                 gui_path = self.path_select.path
                 if gui_path and reason:
+                    if '/' in reason:
+                        raise Exception('File name cannot include reserved character "/".')
```
Comment (mrakitin) [inline startup/99-gui-ht.py:721]: Maybe there can be a more scoped custom exception here?

Such as:
```py
class FileInvalidException(Exception):
    pass
```

The same can be useful for the other 2 exceptions below.

**Review by mrakitin — COMMENTED** [review]
Looks good to me. I have an optional suggestion regarding a custom exception.

File: startup/99-gui-ht.py (hunk 1 above)
Comment (RobertSchaffer1) [reply startup/99-gui-ht.py]: Makes sense, done.

**Pull request merged**